src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceSyncResponse.scala

```diff
@@ -1,11 +1,13 @@
 package com.cognite.sdk.scala.v1.fdm.instances
 
 import io.circe.generic.semiauto.deriveEncoder
-import io.circe.{Decoder, Encoder, HCursor}
+import io.circe.{Decoder, Encoder, HCursor, Json}
 
 final case class InstanceSyncResponse(
     items: Option[Map[String, Seq[InstanceDefinition]]] = None,
-    nextCursor: Option[Map[String, String]] = None
+    nextCursor: Option[Map[String, String]] = None,
+    typing: Option[Map[String, Map[String, Map[String, Map[String, TypePropertyDefinition]]]]] =
```

Comment:
That's a bit too much of unspecified types :joy: 
I think for sdk it makes sense to define typed wrappers that'd give names to those things. Type alias may also work to show which `String` means what but it is less strict

In case top 3 levels of maps aren't going to ever be empty (or if all we ever case about is getting to `TypePropertyDefinition`) that should also be isomorphic to `Map[(String, String, String, String), TypePropertyDefinition]`
Looks like the keys are: `batch_group_name`, `space`, `externalId`, `property`

The absolute minimum is to have comment describing the type

Reply:
yes do not love this, it kind of follows the pattern of InstanceFilterResponse, except it need to look up the Map[String, Map[String, Map[String, TypePropertyDefinition]] based on batch_group_name for each item batch, though usually just 1.